### PR TITLE
Team Retreat Blog Post

### PR DESCRIPTION
**Pull request by ABizzinotto**
@etagwerker @lubc here's a blog post about our team retreat.

### CODE REVIEW

File: _posts/2020-03-07-Team-Retreat.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Team Retreat: How They Can It Help Remote Teams "Click"
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: Typo here: 

```suggestion
title: Team Retreat: How They Can Help Remote Teams "Click"
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+categories: [agile]
+author: abizzinotto
+
+---
+
+Remote work is a growing reality. In the 21st century, we have distributed companies that work 100% remotely, with team members scattered around the world.
+
+While remote work has its benefits - the diversity in background and experiences, access to an immense talent pool, cut down office costs to name a few - it also brings its challenges. And the biggest one is communication.
+
+There are several pieces of software that aim to faciliate communication in a remote work environment. At OmbuLabs, we use [Slack](https://slack.com) for our day-to-day conversation and [Zoom](https://zoom.us/) for our calls. However, as a remote team, we don't get to meet our teammates face to face. We don't have "water cooler" chats or as many opportunities for casual chats as an office environment.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
There are several pieces of software that aim to facilitate communication in a remote work environment. At OmbuLabs, we use [Slack](https://slack.com) for our day-to-day conversations and [Zoom](https://zoom.us/) for our calls. However, as a remote team, we don't get to meet our teammates face to face. We don't have "water cooler" chats or as many opportunities for casual chats as an office environment.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+So how can we bring our team close together? How can we build that team spirit that comes from seeing every one and talking about our casual, social lives? Well, this month we had a team retreat. And it was great.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto Would be interesting if we can add a photo from our team retreat here?

File: _posts/2020-03-07-Team-Retreat.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+<!--more-->
+
+# The Idea
+
+A team retreat is supposed to bring the team together. We are a distributed team, with team mates located in 4 different countries. As such, while some of our team mates are located in the same country / city and have a chance to meet in person every once in a while, folks in different countries only communicate online.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
A team retreat is supposed to bring the team together. We are a distributed team, with teammates located in four different countries. As such, while some of our teammates are located in the same country/city and have a chance to meet in person every once in a while, folks in different countries only communicate online.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+Therefore, it was definitely interesting for us to bring the team together and give everyone an opportunity to interact. After all, we spend a great part of our day in touch. We might as well put a "real world" face to the names.
+
+# The Planning
+
+When putting together a team retreat, it is important to plan it well. What are the best dates? How about a location? Team members will be travelling from their home countries to the retreat destination to spend a few days with the team, it is important to take distance into consideration.
+
+Additionally, it is important to plan for visas. What if a team mate's visa is denied? What if the process takes too long? One lesson we learnt is it is always better to plan ahead. At OmbuLabs, our goal now is to announce the retreat dates and destination a year in advance. This way, the team has a chance to get all necessary documentation sorted.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
Additionally, it is important to plan for visas. What if a teammate's Visa is denied? What if the process takes too long? One lesson we learnt is it is always better to plan ahead. At OmbuLabs, our goal now is to announce the retreat dates and destination a year in advance. This way, the team has a chance to get all necessary documentation sorted.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+# The Agenda
+
+Once the retreat is planned and everything is booked, we need an agenda for the event. After all, we're having a once in a year opportunity to bring everyone to the same location.
+
+We like to open our retreats with a talk from our CEO on the company's vision and mission. This way, we all know where we're going.
+
+It is also important to plan your agenda carefully. A good way to get to great topics is to ask yourself "what challenges are we facing right now? And, of these, which ones impact us the most?". In a team retreat we have the opportunity to bring all heads together in the same room. It is a unique opportunity to foster discussion face-to-face, without the reservations we all have when interacting online. Therefore, make it matter.
+
+At our team retreat, we talked about company culture, discussed our mentorship programme, and gathered ideas to make our systems and processes better.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
At our team retreat, we talked about company culture, discussed our mentorship program, and gathered ideas to improve our systems and processes. 
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- Ask Me Anything
+- Two Truths and a Lie
+
+There is a great deal of flexibility with activities and lots of good games you can play with your team. Take advantage of it and choose games that will bring your team together.
+
+# Rules
+
+It is not all fun and games. It is important to understand the whole team will be together in one location for a company event. Even though we all want to have fun, it is important to keep it professional and make sure nobody is made uncomfortable. 
+
+In order to ensure things went smoothly, we came up with a Code of Conduct that was shared with all team members. This way, we established procedures for team members to follow if they felt uncomfortable for any reason.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto Maybe you can mention that our CoC was heavily influenced by the contributor covenant? https://www.contributor-covenant.org

File: _posts/2020-03-07-Team-Retreat.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+# Feedback
+
+After the retreat, it is good to get feedback.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto Considering this is the first team retreat we organize, I'm sure there will be many opportunities to make things better. 👍

File: _posts/2020-03-07-Team-Retreat.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+# The Activities
+
+When planning activities in a team retreat, it is important to think of the value they bring. We want activities that will bring the team together while being entertaining, that will serve a clear purpose while being fun and enjoyable.
+
+Some activities we did include:
+
+- [SpaceTeam](http://playspaceteam.com)
+- [Escape Room in a Box](http://www.escaperoominabox.com/en-us)
+- Ask Me Anything
+- Two Truths and a Lie
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto Maybe you could include a photo in here?

File: _posts/2020-03-07-Team-Retreat.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+It is not all fun and games. It is important to understand the whole team will be together in one location for a company event. Even though we all want to have fun, it is important to keep it professional and make sure nobody is made uncomfortable. 
+
+In order to ensure things went smoothly, we came up with a Code of Conduct that was shared with all team members. This way, we established procedures for team members to follow if they felt uncomfortable for any reason.
+
+# Feedback
+
+After the retreat, it is good to get feedback. 
+
+We shared a feedback form with the whole team giving people the chance to talk about our choice of destination, the activities and the experience itself. What was amazing? What could we do better? This feedback will help us plan better retreats and grow closer as a company
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto Not sure if you want to mention it, but I think it was positive to include so time off during the retreat. That way the team could connect on a personal level in an unstructured way. For example: I love that the team got together at the beach and shared songs that they liked/disliked :)

File: _posts/2020-03-07-Team-Retreat.md (hunk 3 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: Good point. @lubc could you share the official photo we took on Saturday?

File: _posts/2020-03-07-Team-Retreat.md (hunk 7 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: @etagwerker done.

File: _posts/2020-03-07-Team-Retreat.md (hunk 9 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: Done

File: _posts/2020-03-07-Team-Retreat.md (hunk 8 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: @etagwerker added an extra line on that.

File: _posts/2020-03-07-Team-Retreat.md (hunk 10 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: @etagwerker I added a Time Off section. Could you review?

File: _posts/2020-03-07-Team-Retreat.md (hunk 3 above)
Comment (etagwerker) [reply _posts/2020-03-07-Team-Retreat.md]: @ABizzinotto I think he shared a few in Slack.

File: _posts/2020-03-07-Team-Retreat.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+# Rules
+
+It is not all fun and games. It is important to understand the whole team will be together in one location for a company event. Even though we all want to have fun, it is important to keep it professional and make sure nobody is made uncomfortable. 
+
+In order to ensure things went smoothly, we used the [Contributor Covenant](https://www.contributor-covenant.org/) for inspiration to come up with a Code of Conduct that was shared with all teammembers. This way, we established procedures for team members to follow if they felt uncomfortable for any reason.
+
+# Time Off
+
+We planned the retreat to include some time off during the week so teammembers could interact in an unstructured environment as well as enjoy the location and engage in activities that interest them.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
We planned the retreat to include some time off during the week so team members could interact in an unstructured environment as well as enjoy the location and engage in activities that interest them.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+We were away for a week. On Monday, we were all travelling in, so we didn't include any planned activities. On Tuesday, Wednesday and Thursday we worked on client projects during the morning and company activities in the afternoon. We left evenings out so teammembers could enjoy the resort and interact.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
We were away for a week. On Monday, we were all travelling in, so we didn't include any planned activities. On Tuesday, Wednesday and Thursday we worked on client projects during the morning and company activities in the afternoon. We left evenings out so teammates could enjoy the resort and interact.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+Finally, we took Friday off. This gave everyone a chance to explore, engage on a more personal level and enjoy external activities of interest. 
+
+We found including time off during the retreat to be very positive. It allowed us to get to know each other on a personal level, foster better relationships and we also learnt a lot about each other's taste in music.
+
+# Feedback
+
+After the retreat, it is good to get feedback. This was our first team retrat, so there is definitely room for improvement.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
After the retreat, it is a good idea to get feedback. This was our first team retreat, so there is definitely room for improvement.
```

File: _posts/2020-03-07-Team-Retreat.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+
+# Time Off
+
+We planned the retreat to include some time off during the week so teammembers could interact in an unstructured environment as well as enjoy the location and engage in activities that interest them.
+
+We were away for a week. On Monday, we were all travelling in, so we didn't include any planned activities. On Tuesday, Wednesday and Thursday we worked on client projects during the morning and company activities in the afternoon. We left evenings out so teammembers could enjoy the resort and interact.
+
+Finally, we took Friday off. This gave everyone a chance to explore, engage on a more personal level and enjoy external activities of interest. 
+
+We found including time off during the retreat to be very positive. It allowed us to get to know each other on a personal level, foster better relationships and we also learnt a lot about each other's taste in music.
```
Comment (etagwerker) [inline _posts/2020-03-07-Team-Retreat.md]:
```suggestion
We found including time off during the retreat to be very positive. It allowed us to get to know each other on a personal level, foster better relationships and we also learned a lot about each other's background and taste in music.
```

File: _posts/2020-03-07-Team-Retreat.md (hunk 3 above)
Comment (ABizzinotto) [reply _posts/2020-03-07-Team-Retreat.md]: Added.

File: _posts/2020-03-07-Team-Retreat.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+Once the retreat is planned and everything is booked, we need an agenda for the event. After all, we're having a once in a year opportunity to bring everyone to the same location.
+
+We like to open our retreats with a talk from our CEO on the company's vision and mission. This way, we all know where we're going.
+
+It is also important to plan your agenda carefully. A good way to get to great topics is to ask yourself "what challenges are we facing right now? And, of these, which ones impact us the most?". In a team retreat we have the opportunity to bring all heads together in the same room. It is a unique opportunity to foster discussion face-to-face, without the reservations we all have when interacting online. Therefore, make it matter.
+
+At our team retreat, we talked about company culture, discussed our mentorship programme, and gathered ideas to make our systems and processes better. 
+
+However, a good agenda must also include fun activities. A system we found effective was to intercalate presentations and discussions with fun team activities. This way, the day doesn't get boring and we can foster better discussions.
```
Comment (FionaDL) [inline _posts/2020-03-07-Team-Retreat.md]: consider replacing "intercalate" with "break up"

**Pull request merged**